### ISSUE TITLE
DI test per admin factory

### ISSUE THREAD

**Issue by dbu** [symfony-cmf/sonata-phpcr-admin-integration-bundle#38]
each bundle integration should have its own configuration test.

**Comment by ElectricMaxxx** [comment]
Maybe when we are fine with the complete configuration.

**Comment by ElectricMaxxx** [comment]
This should not be relevant for tagging RC-1

**Comment by ElectricMaxxx** [comment]
This should be an easy pick for somebody whant to get into the code.

**Comment by wouterj** [comment]
I don't think this is an easy pick. Easy picks are issues that can be fixed without having any prior knowledge. This issue require knowledge about both Matthias' package as well as the factories.

**Comment by ElectricMaxxx** [comment]
Should be done after #92.

**Comment by ElectricMaxxx** [comment]
just an nice to have regarding the release.
